CRM/Report/Form/Contribute/Summary.php

```diff
@@ -226,6 +226,24 @@ public function __construct() {
           ),
         ),
       ),
+      'civicrm_financial_trxn' => array(
+        'dao' => 'CRM_Financial_DAO_FinancialTrxn',
+        'fields' => array(
+          'card_type' => array(
+            'title' => ts('Credit Card Type'),
+            'dbAlias' => 'GROUP_CONCAT(financial_trxn_civireport.card_type SEPARATOR ",")',
```

Comment:
this looks safe because there is a default groupBy

    else {
      $groupBy = "{$this->_aliases['civicrm_contact']}.id";
      $this->_groupBy = "GROUP BY {$this->_aliases['civicrm_contact']}.id";
    }